src/Serilog.Sinks.OpenTelemetry/Helpers/OpenTelemetryEnvironment.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Serilog.Sinks.OpenTelemetry;
+
+namespace Serilog.Helpers;
+
+internal static class OpenTelemetryEnvironment
+{
+    public static void Configure(BatchedOpenTelemetrySinkOptions options)
+    {
+        options.Endpoint = Endpoint;
+        options.Headers = Headers;
+        options.ResourceAttributes = ResourceAttributes;
+    }
+
+    public static string Endpoint
+        => Environment.GetEnvironmentVariable("OTEL_EXPORTER_OTLP_ENDPOINT") ?? throw new InvalidOperationException("OTEL_EXPORTER_OTLP_ENDPOINT was not found");
+
+    public static Dictionary<string, string> Headers
+        => GetHeaders(Environment.GetEnvironmentVariable("OTEL_EXPORTER_OTLP_HEADERS"));
+
+    public static Dictionary<string, object> ResourceAttributes
+        => GetResourceAttributes(Environment.GetEnvironmentVariable("OTEL_RESOURCE_ATTRIBUTES"));
+
+    private static Dictionary<string, string> GetHeaders(string? config)
+    {
+        Dictionary<string, string> configs = [];
+        foreach (var part in config?.Split(',') ?? [])
+        {
+            if (part.Split('=') is { Length: 2 } parts)
+                configs.Add(parts[0], parts[1]);
+            else
+                throw new InvalidOperationException($"Invalid header format: {part}");
```

Comment:
Nit: exception messages should end with `.`, although it's a bit awkward here.

Reply:
I've added the header name to make debugging easier.

About the exception here, since the value can come from manual creation, I think that being able to know when the value is not in the expected format can help to fix it asap.

But if you still think that it isn't necessary here, I can remove it.